\documentclass[aspectratio=169,12pt,t]{beamer}
\usepackage{graphicx}
\setbeameroption{hide notes}
\setbeamertemplate{note page}[plain]
\usepackage{listings}
\usepackage{eepic}

\input{header.tex}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
% end of header
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

% title info
\title{general HMM for multi-parent populations}
\author{\vspace*{-10pt} \href{https://kbroman.org}{Karl Broman}}
\institute{Biostatistics \& Medical Informatics, UW{\textendash}Madison}
\date{\href{https://twitter.com/kwbroman}{\tt \scriptsize \color{foreground} @kwbroman}
\\[-4pt]
\href{https://kbroman.org}{\tt \scriptsize \color{foreground} kbroman.org}
\\[-4pt]
\href{https://github.com/kbroman}{\tt \scriptsize \color{foreground} github.com/kbroman}
\\[-4pt]
{\scriptsize \href{https://kbroman.org/Talk_GeneralHMM}{\tt kbroman.org/Talk\_GeneralHMM}}
}


\begin{document}

% title slide
{
\setbeamertemplate{footline}{} % no page number here
\frame{
  \titlepage

  \vfill \hfill
  \href{https://creativecommons.org/publicdomain/zero/1.0/}{\includegraphics[height=7mm]{Figs/cc-zero.png}}

  \note{
    These are slides for a talk for the CTC ({\tt
    www.complextrait.org/ctc2021/}) on 1 Sept 2021.

    Slides:
    \href{https://kbroman.org/Talk_GeneralHMM/general_hmm.pdf}{\tt
      kbroman.org/Talk\_GeneralHMM/general\_hmm.pdf} \\[12pt]
    Slides with notes:
    \href{https://kbroman.org/Talk_GeneralHMM/general_hmm_notes.pdf}{\tt
      kbroman.org/Talk\_GeneralHMM/general\_hmm\_notes.pdf} \\[12pt]
    Source: \href{https://github.com/kbroman/Talk_GeneralHMM}{\tt github.com/kbroman/Talk\_GeneralHMM}
    \\[24pt]
    Related paper on bioRxiv: \href{https://doi.org/gswx}{\tt doi.org/gswx}
  }

} }




\begin{frame}[c]{}

  \begin{columns}

    \column{0.5\textwidth}

    \centering
    {\color{title} Recombinant Inbred Lines}

    \bigskip

    \figh{Figs/ri8.pdf}{0.8}


    \column{0.5\textwidth}

    \centering
    {\color{title} Advanced Intercross Population}

    \bigskip

    \figh{Figs/hs.pdf}{0.8}

  \end{columns}


  \note{
    Multi-parent populations are experimental crosses starting from
    multiple inbred founder lines.

    Major examples include the Collaborative Cross, a set of 8-way
    recombinant inbred lines in mouse, and Hetereogeneous Stock, which
    have been developed in both mice and rats and are advanced
    intercross populations derived from 8 founders. The Diversity
    Outbred mouse population is similar to HS. In plants, multi-parent
    recombinant inbred lines are called MAGIC lines (for multiparent
    advanced generation inter-cross).

    The offspring chromosomes will be mosaics of the founder
    chromosomes. Multi-parent populations can be homozygous (like RIL)
    or heterozygous (like HS). The number of founders need not be 8.
  }

\end{frame}




\begin{frame}[c]{Genome reconstruction}


  \figh{Figs/genome_reconstr.pdf}{0.9}

  \note{

    A key step in the analysis of multi-parent populations is genome
    reconstruction: using dense SNP genotypes in the founders and MPP
    offspring to infer the haplotypes across the genome.

    Here we consider a 1 Mbp region on chromosome 14 in a single
    Diversity Outbred Mouse. Open and closed circles indicate AA and
    BB genotypes at SNPs. Gray circles indicate AB heterozygous
    genotypes. Using the SNP data along the chromosome, we can
    calculate the probability of each possible genotype at each
    position.

    For this mouse, the left half of the interval looks to be
    homozygous DD, while the right half looks to be heterozygous AD.
  }

\end{frame}



\begin{frame}[c]{QTL genome scan}


  \only<1|handout 0>{\figh{Figs/qtl_scan_snps.png}{0.85}}
  \only<2>{\figh{Figs/qtl_scan.png}{0.85}}

  \note{

    One could skip the whole genome reconstruction and just do QTL
    analysis at the SNPs, as is done in GWAS.
    If the genotyped SNPs include individual causal polymorphisms,
    this could be best.

    But if there are multiple causal polymorphisms in a region QTL
    analysis with the inferred haplotypes may be more powerful.
    Moreover, if the founder strains have been sequenced, you can use
    the reconstructed genomes to get inferred genotypes at all
    polymorphisms in the founders. (Similar approaches were used in
    human GWAS, based on HapMap SNPs.)

    Here, the single-SNP analysis shows significant evidence for a
    single QTL on chromosome 1. The haplotype analysis indicates
    evidence for a second QTL on chromosome 4.

    Beyond QTL mapping, genome reconstructions are useful in data
    diagnostics. For example, the estimated number of crossovers is
    useful when assessing sample quality.
  }

\end{frame}



\begin{frame}[c]{DO genome}


  \figh{Figs/do_genome.pdf}{0.85}

  \note{
    Here is the reconstructed genome of a Diversity
    Outbred mouse. (The white segments are undetermined.)

    Our goal is to figure this out, using SNP genotypes on this mouse
    plus the 8 founder lines.
  }

\end{frame}





\begin{frame}{Hidden Markov model}

\figw{Figs/hmm.pdf}{1.0}

\bigskip

{
\centering
\renewcommand{\arraystretch}{2.0}

\begin{tabular}{l@{\hspace{1cm}}l}
Initial    & $\text{Pr}(G_1 = g)$ \\
Transition & $\text{Pr}(G_{i+1} = g' \ | \ G_i = g)$ \\
Emission   & $\text{Pr}(O_i \ | \ G_i = g)$
\end{tabular}

}



  \note{
    The main approach for genome reconstruction is to use a hidden
    Markov model. The underlying diplotypes we're trying to determine
    follow a Markov chain $\{G_i\}$, but are unobserved. We observe
    SNP genotypes $\{O_i\}$, with an assumed conditional independence
    structure, where given $G_i$, $O_i$ is conditionally independent
    of everything else.
  }

\end{frame}





\begin{frame}[c]{Exact probabilities}

  \note{
  }

\end{frame}





\begin{frame}[c]{Generic model}

  \begin{columns}

    \column{0.5\textwidth}

    \figh{Figs/founder_pop.pdf}{0.5}


    \column{0.5\textwidth}

    $k$ founders in proportions $\{\alpha_i\}$ \\[4pt]
    $n$ generations of random mating \\[18pt]
    {\color{title} Random chromosome}: \\[4pt]
    \qquad $\pi_i = \alpha_i$ \\[4pt]
    \qquad $t_{ij} = \alpha_j \ [\text{1} - (\text{1} - r)^n] \quad
    \text{when } i \ne j$ \\[18pt]
    {\color{title} Map expansion:} \\[4pt]
    \qquad $n (\text{1}-\sum\alpha_i^{\text{2}})$ \\[4pt]
    \qquad $= n \left(\frac{k-\text{1}}{k}\right) \quad \text{if } \alpha_i \equiv \text{1}/k$


  \end{columns}


  \note{
  }

\end{frame}






\begin{frame}[c]{DO application}

  \note{
  }

\end{frame}





\begin{frame}[c]{X chr in CC}

  \note{
  }

\end{frame}





\begin{frame}[c]{X chr reconstruction}

  \note{
  }

\end{frame}





\begin{frame}[c]{Summary}

  \bbi

\item Generic model for genome reconstruction in multi-parent
  populations

\item Specific relative proportions of founders + effective number of
  generations of random mating

\item Basic conclusion: {\hilit HAPPY is effective}

\item bioRxiv manuscript:
  \href{https://doi.org/gswx}{\tt \lolit doi.org/gswx}


  \ei

  \note{
    It's always good to provide a summary.
  }

\end{frame}



\begin{frame}[c]{}

\Large

Slides: \href{https://kbroman.org/Talk_GeneralHMM}{\tt kbroman.org/Talk\_GeneralHMM}

\vspace*{-7mm}
\hfill
\href{https://creativecommons.org/publicdomain/zero/1.0/}{\includegraphics[height=7mm]{Figs/cc-zero.png}}

\vspace{3mm}

bioRxiv manuscript:
\href{https://doi.org/gswx}{\tt \lolit doi.org/gswx}

\vspace{4mm}

\href{https://kbroman.org}{\tt \lolit kbroman.org}

\vspace{4mm}

\href{https://github.com/kbroman}{\tt \lolit github.com/kbroman}

\vspace{4mm}

\href{https://twitter.com/kwbroman}{\tt \lolit @kwbroman}

\vspace{4mm}

\href{https://kbroman.org/qtl2}{\tt \lolit kbroman.org/qtl2}




\note{
  Here's where you can find me and these slides.
}

\end{frame}






\end{document}
